### PR TITLE
added picter-labels.json

### PR DESCRIPTION
**Pull request by mattiloh**
I just ported the labels we defined on Gitlab, back then.

Here's a screenshot to see the color schema:

labels picter gitlab

### CODE REVIEW

**Comment by lhrnd0** [comment]
I would remove some of them since now we have our ZenHub board. What do you think?

**Comment by mattiloh** [comment]
Sure! Just let me know which ones 😉

**Comment by lhrnd0** [comment]
### I would remove 

- `Doing`
- `Merge request`
- `To Do`
- `status: review needed`
- `status: ready for merge`

### Propose to add

- `type: component` not sure because this could be a feature also

### Propose to merge

- `type: question` with `type: discussion`, they seem to me the same thing

**Comment by mattiloh** [comment]
Thanks @lcsrinaldi! The screenshot was rather meant to show the colors, I already removed most of the labels you suggested for removal. Just `status: review needed` is still existing. But you're right, we can remove it, since we have a kanban board now. What do you think about a `status: wip` label? This could be added to PRs, that are still wip...

I'm not sure about merging `question` and `discussion`. This can be problematic especially for open-source projects.

**Comment by noxan** [comment]
Label `status: wip` can also be done with the columns of zenhub. If we use `In progress` for the PR, it is clear what it's about.

File: picter-labels.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    "name": "status: blocked",
+    "color": "000000"
+  },
+  {
+    "name": "status: help wanted",
+    "color": "000000",
+    "formerNames": ["help wanted"]
+  },
+  {
+    "name": "status: review needed",
```
Comment (noxan) [inline picter-labels.json]: Not so happy with this one. Either a PR is ready to be reviewed or it is in progress. To manage this manually with a label is sure to be outdated.

**Comment by lhrnd0** [comment]
I don't know about `status: wip`, I don't see much benefit. 

About merging those two, can you explain what's the difference between them? In the end they are so much alike.

**Comment by mattiloh** [comment]
Ok, you're probably right, @lcsrinaldi. I would favor `type: question` over `type: discussion` then, since it seems to be more generic.

**Comment by lhrnd0** [comment]
Up to you then :) thanks!

**Pull request merged**